EDU-13253 - Review: 6. Building your own store theme (#1612)
vtexdocs/dev-portal-content: docs/guides/vtex-io/Getting-Started/getting-started-3/vtex-io-documentation-6-buildingyourownstoretheme.md

hidden: false
category: "Storefront Development"
createdAt: "2020-08-11T07:03:17.864Z"
updatedAt: "2024-11-28T20:06:03.731Z"
excerpt: "Explore resources to personalize your VTEX store with Store Framework."
---

> After making the first setups, creating your Store Theme project, and learning how to customize your store pages, you can begin personalizing your store more advancedly.

Reviewer: ```suggestion
After completing the initial setup, creating your Store Theme project, and learning how to customize your store pages, you can begin advanced personalization of your store.
```

Under the [Store Framework](https://developers.vtex.com/docs/guides/store-framework) category, you will find a set of guides to help you customize your storefront:

- [Building pages](https://developers.vtex.com/docs/guides/store-framework-building-pages): Build various types of pages tailored to your business needs.
- [Using components](https://developers.vtex.com/docs/guides/store-framework-using-components): Build your storefront, from basic elements to advanced customizations, by leveraging VTEX IO Store Framework components.
- [Working with Site Editor](https://developers.vtex.com/docs/guides/store-framework-working-with-site-editor): Customize your Store Framework storefront by using Site Editor.
- [Styling](https://developers.vtex.com/docs/guides/store-framework-styling): Customize your store layout with Store Framework.

You can also access our [hands-on course on the VTEX Store Framework](https://learn.vtex.com/page/learning-path-lang-en).

> > ℹ Once you’ve finished customizing your storefront, whether by completing the hands-on course or following our guides, keep in mind that your Store Theme app is still stored locally on your computer and it is linked to your VTEX store only through the development workspace. To make your changes visible to end users, you need to [make your theme content public](https://developers.vtex.com/docs/guides/vtex-io-documentation-making-your-theme-content-public).

Reviewer: Talvez faça sentido mencionar essa doc aqui: https://developers.vtex.com/docs/guides/vtex-io-documentation-workspaces-best-practices

Tínhamos que incorporar melhor esse conteúdo, eu acho 🤔 
Fala para subir num repo do Github para que outros devs possam ter mais visibilidade do que está sendo feito, versões do tema, etc.
